leme-onboarding/src/components/Validation.tsx

```diff
@@ -0,0 +1,87 @@
+
+
+export function validationEmail(email: string) {
+    /**
+     * 0 - Validated Email
+     * 1 - Empty field
+     * 2 Email not validated
+    */
+
+    if (email.length === 0) {
+        return 1;
+    }
+
+    const emailRegex = /^[a-zA-Z0-9]+@(?:[a-zA-Z0-9]+\.)+[A-Za-z]+$/;
+
+    if (emailRegex.test(String(email).toLowerCase())) {
+        return 0;
+    }
+    else {
+        return 2;
+    }
+}
+
+export function validationPassword(password: string) {
+    /*
+        * 0 Validated Password
+        * 1 Empty
+        * 2 Minimum 7 characters
+        * 3 One digit e one number 
+        */
```

Comment:
Se você deseja colocar algo como uma descrição de uma função, classe ou método, recomendo usar no modelo do [jsdoc](https://jsdoc.app/)
Nesse caso ficaria assim:
```suggestion

/**
 * Validates a password
 * 
 * @param {string} password
 * @return {number}
 * - 0 Validated Password
 * - 1 Empty
 * - 2 Minimum 7 characters
 * - 3 One digit e one number 
 */
export function validationPassword(password: string) {
```

Nesses moldes a documentação pode ser escrita usando markdown
Ai olha como o vscode consegue ver essa documentação, e todo mundo que for usar sua função depois
<img width="381" alt="Screen Shot 2020-05-11 at 09 19 04" src="https://user-images.githubusercontent.com/36424331/81560808-8118f600-9368-11ea-821c-aa053d9ddf6e.png">

Reply:
Que legal! Não sabia que dava pra escrever em markdown

Resolvido!